frontend/src/metabase/admin/permissions/selectors/data-permissions/data-sidebar.ts

```diff
@@ -51,6 +53,7 @@ const getTableId = (id: string | number) => `table:${id}`;
 const getDatabasesSidebar = (metadata: Metadata): DataSidebarProps => {
   const entities = metadata
     .databasesList({ savedQuestions: false })
+    .filter(db => !PLUGIN_AUDIT.isAuditDb(db as DatabaseType))
```

Comment:
this isn't the _best_ way to handle this. `databasesList` is deprecated, and if we use the RTK query cache, it would just work correctly since the databases endpoint excludes the analytics DB. _However_ the permissions editor is generated by a whole bunch of selectors that would have to be completely refactored to be able to use RTK in a sane way.